docs/platform.md

```diff
@@ -8,20 +8,76 @@ images: ["/platform/build.svg", "/services/ml/configure.svg"]
 {{< board-carousel >}}
 <br>
 
+The open-source executable binary that runs on a Viam-powered {{< glossary_tooltip term_id="machine" text="machine" >}} is called `viam-server`.
+`viam-server` runs and manages everything on your machine, including communications between hardware, software, and the cloud.
+
+## What a machine is in Viam
+
+A {{< glossary_tooltip term_id="machine" text="machine" >}} is any computer (SBC, server) running [`viam-server`](/architecture/#viam-server-and-the-micro-rdk) or any microcontroller running `viam-micro-server`, plus any hardware attached to that computer.
+
+## How machines are structured, configured and organized
+
+Learn more about the structure and configuration of machines:
+
+|                        | Structure...                            | Configuration...                     |
+| ---------------------- | --------------------------------------- | ------------------------------------ |
+| ...of one machine ->   | [Architecture](/architecture/)          | [Machine Configuration](/configure/) |
+| ...of many machines -> | [Cloud Organization Hierarchy](/cloud/) | [Deploy a Large Fleet](/fleet/)      |
+
+## The things that make up a machine
+
+On your machine, you configure components and services.
+Some are built-in, and many more are available in the [registry](/registry/).
+You can also write your own and add them to the registry.
+
+On your machine, you can configure the following components:
+{{< cards >}}
+{{% relatedcard link="/components/arm" %}}
+{{% relatedcard link="/components/base" %}}
+{{% relatedcard link="/components/board" %}}
+{{% relatedcard link="/components/camera" %}}
+{{% relatedcard link="/components/encoder" %}}
+{{% relatedcard link="/components/gantry" %}}
+{{% relatedcard link="/components/generic" %}}
+{{% relatedcard link="/components/gripper" %}}
+{{% relatedcard link="/components/input-controller" %}}
+{{% relatedcard link="/components/motor" %}}
+{{% relatedcard link="/components/movement-sensor" %}}
+{{% relatedcard link="/components/power-sensor" %}}
+{{% relatedcard link="/components/sensor" %}}
+{{% relatedcard link="/components/servo" %}}
+{{< /cards >}}
+And you can use the following platform capabilities and services to make your machines smarter and better:
+{{< cards >}}
+{{% relatedcard link="/services/data" %}}
+{{% relatedcard link="/services/ml/deploy" alt_title="Machine Learning" %}}
+{{% relatedcard link="/services/motion" %}}
+{{% relatedcard link="/services/navigation" %}}
+{{% relatedcard link="/services/slam" %}}
+{{% relatedcard link="/services/vision" %}}
+{{< /cards >}}
+
 {{< cards >}}
-{{% card link="/architecture/" %}}
-{{% card link="/configure/" %}}
-{{% card link="/fleet/" %}}
-{{% card link="/cloud/" %}}
-{{% card link="/components/" %}}
-{{% card link="/services/" %}}
 {{% card link="/registry/" %}}
+{{< /cards >}}
```

Comment:
I think just the text link is fine. Given that all component pages will immediately also tell people about the registry I think that's probably enough
```suggestion
```